### PR TITLE
pass privileged: false through from binci.yml

### PR DESCRIPTION
**Pull request by brandonros**
part of https://github.com/binci/binci/issues/112

### CODE REVIEW

**Comment by brandonros** [comment]
@TomFrost @Fluidbyte are either one of you cool with merging this then doing a minor release please whenever you have time?

File: src/services.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const services = {
       name,
       persist: value.persist || false,
       stopTimeSecs: services.getStopTimeSecs(cfg, value),
-      args: command.get(_.merge(value, { rmOnShutdown: cfg.rmOnShutdown }), name, null)
+      args: command.get(_.merge(value, { rmOnShutdown: cfg.rmOnShutdown, privileged: cfg.privileged }), name, null)
```
Comment (Fluidbyte) [inline src/services.js:73]: this would use `false` by default if no `privileged` prop is in the config.

**Review by Fluidbyte — CHANGES_REQUESTED** [review]

File: src/services.js (hunk 1 above)
Comment (brandonros) [reply src/services.js]: 4da55e2b6313768c9c5426f5c22bffeeac9a747a how's thtat

**Review by Fluidbyte — APPROVED** [review]

**Comment by Fluidbyte** [comment]
@brandonros all merged up, release 5.9.0 is now out. Thanks for adding this.

**Pull request merged**